### ISSUE TITLE
Move main docs to 6-dev branch  for docs.plone.org

### ISSUE THREAD

**Issue by jensens** [plone/plone-backend#46]
The README here lands on Docker Hub, so it should contain  basic about the image and Plone.
In depth information has to be moved to docs.plone.org (6-dev branch)
@stevepiercy knows better where to put it there. Lets keep this issue open until this is finished.

**Comment by stevepiercy** [comment]
To clarify, the only docs I see in this repo are in `README.md`. Is that correct?

Does in-depth information exist somewhere? If so, where?

As far as where in-depth docs of backend should go, anyone can [create a pull request against the `6-dev` branch](https://github.com/plone/documentation/compare/6-dev...my_branch). There is a subdirectory [`docs/backend`](https://github.com/plone/documentation/tree/6-dev/docs/backend) with a draft structure awaiting content.

**Comment by jensens** [comment]
I think we need some kind of deployment documentation, so there will be.

**Comment by jensens** [comment]
we have the deployment training now, this can be closed

**Comment by stevepiercy** [comment]
No, Training is not the authoritative source for Documentation. We maintain Documentation all the time now, whereas a Training is usually only updated by the author. Additionally we do not want to punish users to go between two websites to get the information they seek. Please reopen.

**Comment by jensens** [comment]
I will change the ticket title

**Comment by stevepiercy** [comment]
@jensens for some reason, I cannot assign this ticket to myself or add it to the Plone 6 Docs release GitHub Project. I could not reopen the ticket myself yesterday either. Would you please check my permissions on this repository, and grant me sufficient rights? Thank you!

After I have the opportunity to run through the training and provide feedback in a couple of weeks, then I'll work on this ticket.

**Comment by jensens** [comment]
@stevepiercy indeed, Developers group had only read access. @ericof by intend? Anyway, I change this to "Triage".

**Comment by stevepiercy** [comment]
@jensens thanks. Assigned and added to the GitHub project Release Plone 6 docs.
